deploy/docker/cp-tools/research/cellprofiler-web-api/app/src/hcs_manager.py

```diff
@@ -5,6 +5,17 @@
 import glob
 import os
 from .z_planes_pipeline import ZPlanesPipeline
+from tifffile import imread
+import numpy as np
+from moviepy.editor import ImageSequenceClip
+import time
+from PIL import Image, ImageOps
+import xml.etree.ElementTree as ET
+import json
+import errno
+
+CELLPROFILER_API_COMMON_RESULTS_DIR = os.getenv('CELLPROFILER_API_COMMON_RESULTS_DIR')
+HCS_CLOUD_FILES_SCHEMA = os.getenv('HCS_PARSING_CLOUD_FILES_SCHEMA', 's3')
```

Comment:
May be it would be better to rename this parameter with `CELLPROFILER_API` prefix? (to make all envvars with the same style)